# Requirement for v1.21.1 has nothing to do with program capabilities, see https://github.com/anchore/grype/pull/1484
diff --git a/go.mod b/go.mod
index 3d72c07fa9..efdc329286 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/anchore/grype
 
-go 1.21.1
+go 1.21.0
 
 require (
 	github.com/CycloneDX/cyclonedx-go v0.7.1
